website/pages/home/img/logos/consul.svg

```diff
@@ -1 +1,10 @@
-<svg width="54" height="54" viewBox="0 0 54 54" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="26.466" cy="27.251" fill="var(--gray-4, #bfbfc0)" r="4.458"/><circle cx="35.141" cy="27.242" fill="var(--gray-4, #bfbfc0)" r="2.057"/><circle cx="40.725" cy="23.944" r="2.057" fill="var(--gray-4, #bfbfc0)"/><circle cx="43.658" cy="17.453" fill="var(--gray-4, #bfbfc0)" r="2.057"/><circle cx="46.542" cy="23.859" r="2.021" fill="var(--gray-4, #bfbfc0)"/><circle cx="40.746" cy="30.7" r="1.996" fill="var(--gray-4, #bfbfc0)"/><circle cx="46.668" cy="30.535" r="1.91" fill="var(--gray-4, #bfbfc0)"/><circle cx="43.582" cy="37.117" fill="var(--gray-4, #bfbfc0)" r="2.082"/><path d="M26.549 47.952a20.65 20.65 0 1112.572-37.06L36.6 14.15a16.507 16.507 0 100 26.233l2.521 3.329a20.461 20.461 0 01-12.573 4.24z" fill="var(--gray-4, #bfbfc0)"/></svg>
\ No newline at end of file
+<svg width="144" height="144" viewBox="0 0 144 144" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Opted to directly manipulate this svg logo here because the fill is a different color than what we have in mktg-logos and this is being converted to an image.